### DISCUSSION TITLE
2023年2月12日AWS线下活动现场或线上参与问题入口

### DISCUSSION THREAD

**Discussion by zhuo-yoyowz** [openvinotoolkit/openvino_notebooks#817] [Show and tell]
AI技术进入加速发展阶段，现阶段引爆全球的AIGC技术，包括chatGPT的出现会让你产生紧张感/焦虑吗？你担心AI代替你的工作吗？你认为如何可以在AI高速发展的时代保持自己的职场竞争力？

**Comment by jeffreymingyue** [comment]
不担心

**Reply by charliezhang76** [reply]
AI可以替代一部分人，但是无法取代使用AI的人。
所以不断提升自我，驾驭技术。

**Comment by curiousRay** [comment]
不担心。挑战也是新的机遇。

**Comment by Githubxinwei** [comment]
不会被完全替代吧。

**Comment by 2023chatgpt** [comment]
在未来人工智能终将代替人类。

**Comment by april-lzy** [comment]
确实会替代一些工作，但不担心会被完全替代，个人认为现实问题和环境的高复杂度AI还很难做到高准确率和高效率地处理与解决

**Comment by wanghaoyu0210** [comment]
不担心，因为老板自己都搞不清楚需求。

**Comment by gtsolo** [comment]
应该是一个很好的辅助工具！

**Comment by yzpower** [comment]
AI技术进入加速发展阶段，现阶段引爆全球的AIGC技术，包括chatGPT的出现会让你产生紧张感/焦虑吗？
会！

你担心AI代替你的工作吗？
会！重复性的工作很容易被代替。

你认为如何可以在AI高速发展的时代保持自己的职场竞争力？
从事更创造性的工作。

**Comment by yuliang119110** [comment]
不会的，人还是很强的

**Comment by piedro** [comment]
产业链会被重构

**Comment by yuliang119110** [comment]
不会的，人工智能和要人要互相匹配

**Comment by MagnetStone** [comment]
不担心，AI不会取代人，但很有可能你被掌握AI(如现在火热的chatGPT)的人取代，加油吧，打工人！

**Comment by zhou7rui** [comment]
不担心  具体的需求千变万化。持续学习，了解它并掌握它。

**Comment by zjwinner1019** [comment]
不会被替代，人类总不见得变成一颗肉球一个大脑吧。

**Comment by zpeng1989** [comment]
```
作为一个AI模型，我没有感情和意识，因此不会产生紧张感或焦虑。我的目的是帮助用户更好地了解信息和解决问题，而不是代替人类工作。
虽然AI技术的快速发展可能导致一些工作的自动化，但这也会带来新的工作机会和需求。对于保持自己的职场竞争力，建议以下几点：
1.持续学习：保持更新最新技术和知识，以确保自己始终处于最前沿。
2.培养多样性的技能：不依赖于特定技能或技术，而是培养多种技能，以适应不断变化的工作环境。
3.注重人际关系：建立良好的人际关系，对同事和上司建立信任和合作关系，以便在需要帮助时得到支持。
4.关注客户需求：了解客户需求和期望，并努力满足这些需求，以保持自己的价值。
通过努力做到这些，您可以在AI快速发展的时代保持自己的职场竞争力。
```
来自ChatGPT的回答。

**Comment by caswww** [comment]
作为一名程序员，我并不担心AI会完全代替我的工作。毋庸置疑ChatGPT之类的大模型的确在代码生成领域展现出了令人惊叹的能力， 但我认为代码编写任务的难点之一在于Bug的定位的修复，目前看来ChatGPT并没有表现出这方面的能力。 不过ChatGPT一定会提升程序员的编程效率，未来可能被ChatGPT替代的不是程序员这个岗位，而是不会合理利用ChatGPT的程序员。

**Comment by belllee** [comment]
不担心，AI只能取代一些初级程序员。只要持续保持自己的竞争力，就不怕被取代。

**Comment by STORKP** [comment]
不会，gpt会是一个更好用的工具，解放生产力，好比算盘和计算机的区别

**Comment by stingchen** [comment]
会替代一些工作内容，但不能替代完整的工作职位。反而增强了工作能力，提高了效率，让人集中在更有创意的工作上

**Comment by yuzq3721** [comment]
AI技术进入加速发展阶段，现阶段引爆全球的AIGC技术，包括chatGPT的出现会让你产生紧张感/焦虑吗？你担心AI代替你的工作吗？你认为如何可以在AI高速发展的时代保持自己的职场竞争力？

重复性工作会被完全替代，低创造性工作可能部分替代，高创新工作才有竞争力！

**Comment by GameBoyPlaying** [comment]
相比现在商业的主流的ToB端的像康耐视这些都传统的视觉算法公司有成熟的解决方案，并深度定制一些行业.
1.英特尔该款产品在易用性
2.云平台是否可以迁移到到用户的内网，有些主流的车厂 都有自己的数据库中还有内网，是否可以部署用户的内网服务器？
3.有一些典型的场景 case 介绍吗？

**Comment by mgl0919** [comment]
有焦虑,担心AI代替工作,AI图像识别替代保安监控;自然语言处理等我们的写手博主都会没粉;...
还是要加强人与人直接接触的转变和不断学习提升学习能力,
应对延迟退休和AI海啸的冲击.

**Reply by mgl0919** [reply]
不在现场

**Comment by cwmgqq** [comment]
请问老师openmiro是否可以支持大家的工作 哪种最有优势

**Reply by zhuo-yoyowz** [reply]
我们这款工具套件的名称是OpenVINO :) 。OpenVINO可以支持大家优化深度学习模型的优化与推理的加速，并方便大家在多种平台部署使用深度学习模型。欢迎到我们的网址openvino.ai了解更为详细的信息，当然也可以直接clone我们这个openvino_notebooks的所有代码，快速上手体会我们OpenVINO如何使用，效果如何。

**Comment by Heffie199** [comment]
产业的细分会更多，需要去创新idea
